app/pb_kits/playbook/pb_user/user.rb

```diff
@@ -40,6 +41,13 @@ def avatar_size
       def title_size
         size == "lg" ? 3 : 4
       end
+
+      def show_territory
+        if !territory.blank?
```

Comment:
We can make this more straight forward using the inverse of ‘blank?’.

```suggestion
        if territory.present?
```